### PR TITLE
Adding Blender tutorial video and documentation on NeRF in NeRF composite

### PR DESCRIPTION
**Pull request by cvachha**
Updated Blender Add-on documentation to include a video tutorial and added documentation on compositing NeRF objects in NeRF scenes.

### CODE REVIEW

File: docs/extensions/blender_addon.md [hunk 1]
```diff
@@ -1,40 +1,40 @@
...
 
 4. The add-on should now be installed in the `Render Properties` panel
-
-   ```{image} imgs/blender_tab.png
-   :align: center
-   :width: 400
-   ```
+    <center>
+   <img width="300" alt="image" src="https://user-images.githubusercontent.com/9502341/232202091-c13c66c4-f119-4f15-aa3e-3bf736371821.png">
+   </center>
```
Comment (tancik) [inline docs/extensions/blender_addon.md:29]: Can you delete these image from the repo

File: docs/extensions/blender_addon.md [hunk 2]
```diff
@@ -44,12 +44,12 @@ This Blender add-on allows for compositing with a Nerfstudio render as a backgro
 
    - In object properties, select "Shadow Catcher". This makes the representation invisible in the render, but all shadows cast on it will render. You may have to switch to the cycles renderer to see the shadow catcher option.
    <center>
-   <img width="300" alt="image" src="https://user-images.githubusercontent.com/9502341/211244787-859ca9b5-6ba2-4056-aaf2-c89fc6370c2a.png">
+   <img width="200" alt="image" src="https://user-images.githubusercontent.com/9502341/211244787-859ca9b5-6ba2-4056-aaf2-c89fc6370c2a.png">
```
Comment (tancik) [inline docs/extensions/blender_addon.md:47]: Any reason to have these images smaller? I feel like they are difficult to read 
image

File: docs/extensions/blender_addon.md (hunk 1 above)
Comment (cvachha) [reply docs/extensions/blender_addon.md:29]: Fixed in new commit

File: docs/extensions/blender_addon.md (hunk 2 above)
Comment (cvachha) [reply docs/extensions/blender_addon.md]: Fixed in new commit

**Review by tancik — APPROVED** [review]
LGTM, thanks for adding the tutorial video

**Pull request merged**